docker/README.md

```diff
@@ -1,10 +1,12 @@
-# NVIDIA Merlin Dockerfiles
+# NVIDIA Merlin Docker Containers
 
-We built and host dockerfiles for NVIDIA Merlin on [NGC](https://ngc.nvidia.com/catalog/containers/). The containers can be pulled and started out of the box. In this repository, we provide the Dockerfiles used for the builts. Users can clone and adjust the containers, if necessary.
+All NVIDIA Merlin components are available as open source projects. However, a more convenient way to make use of these components is by using our Merlin NGC containers. We have created Docker containers for NVIDIA Merlin that are hosted on [NGC](https://ngc.nvidia.com/catalog/containers/). 
 
-| Container name             | Dockerfile       | Container location                                                             | Functionality                                         |
+Containers allow you to package your software application, libraries, dependencies, and runtime compilers in a self-contained environment. These containers can be pulled and launched right out of the box. You can clone and adjust these containers if necessary. Here's a list of Docker containers that we currently offer:
+
+| Container Name             | Dockerfile       | Container Location                                                             | Functionality                                         |
 |----------------------------|------------------|--------------------------------------------------------------------------------|-------------------------------------------------------|
 | Merlin-training            | dockerfile.ctr   |  https://ngc.nvidia.com/containers/nvstaging:merlin:merlin-training            | NVTabular and HugeCTR                                 |
```

Comment:
We should fix the urls - this is pointing to the private nvstaging ngc location, and should go to nvidia
```suggestion
| Merlin-training            | dockerfile.ctr   |  https://ngc.nvidia.com/containers/nvidia:merlin:merlin-training            | NVTabular and HugeCTR                                 |
```